README.md

```diff
@@ -146,33 +73,63 @@ Or install via [gem](http://rubygems.org/)
 gem install shopify_api
 ```
 
-### Getting Started
+## Getting Started
+
+ShopifyAPI sessions need to be configured with a fully authorized URL of a particular store before they can start making API calls. To obtain that URL, you can follow these steps:
+
+### 1) Create an app
+
+First, create a new application in either the partners admin or your store admin. 
+
+**Private apps** are used for merchant-owned scripts and apps that run silently in the background on a single shop. Private apps aren't able to render any content in the admin. Private apps are created through the store admin.
 
-ShopifyAPI uses ActiveResource to communicate with the REST web service. ActiveResource has to be configured with a fully authorized URL of a particular store first. To obtain that URL you can follow these steps:
+**Custom apps** are also used for a single shop, but they have access to [app extensions](https://shopify.dev/docs/app-extensions) that allow the app to render content in the admin and are managed and created through the partners dashboard.
 
-1. First create a new application in either the partners admin or your store admin. For a private App you'll need the API_KEY and the PASSWORD otherwise you'll need the API_KEY and SHARED_SECRET.
+**Public apps** can be installed on many stores, and can be added to the Shopify App Store to generate revenue for the developer. 
 
-   If you're not sure how to create a new application in the partner/store admin and/or if you're not sure how to generate the required credentials, you can [read the related shopify docs](https://docs.shopify.com/api/guides/api-credentials) on the same.
+For a private app, you'll need the API_KEY and the PASSWORD; otherwise, you'll need the API_KEY and SHARED_SECRET.
 
-2. For a private App you just need to set the base site url as follows:
+   If you're not sure how to create a new application in the partner/store admin or if you're not sure how to generate the required credentials, you can [read the related Shopify docs](https://docs.shopify.com/api/guides/api-credentials) on the same.
```

Comment:
Update url to shopify.dev

Reply:
https://shopify.dev/tutorials/authenticate-a-private-app-with-shopify-admin

Reply:
This isn't the same guide, I'll have to go back and rewrite the section to account for the change.

Reply:
Done!